added CSS code to design UI of Potential Matches page

jj7lin opened the pull request:
added PPFriends logo, button colors, navigation bar, and background color to the Potential Matches page.

jj7lin committed:
added CSS code to design UI of Potential Matches page

added PPFriends logo, button colors, navigation bar, and background color to the Potential Matches page

chalynpai commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,107 @@
+}
+
+li a:hover:not(.active) {
+  background-color: rgb(
```
you can put this in one line and not in 3

jj7lin committed:
reformatted code

reformatted code as per Chalyn's request

jj7lin committed:
Revert "reformatted code"

This reverts commit 754fe7b9191b78f3ee839f6fe4937adcfa7260c4.

jj7lin committed:
reformatted code

reformatted code as per Chalyn's request

mnsison commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,107 @@
+  border-radius: 12px;
+}
+
+.letsTalkButton {
+  /*"yes, let's match" button */
+  background-color: #e96a6a;
+  border: 2px solid black;
+  width: 240px;
+  color: white;
+  padding: 10px 35px;
+  text-align: center;
+  font-size: 20px;
+  position: relative;
+  margin: 1px 1px;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.notInterestedButton {
+  /*"no, reject match" button*/
+  background-color: #3680a3;
+  border: 2px solid black;
+  width: 240px;
+  color: white;
+  padding: 10px 35px;
+  text-align: center;
+  font-size: 20px;
+  position: relative;
+  margin: 1px 1px;
+  text-decoration: none;
+  cursor: pointer;
+}
```
The properties of these two buttons seem to be the same except the background colour. Do you think it would be better to make a "general" button that will have these similar properties to reduce repetition of code? Then .letsTalkButton and .notInterestedButton will only have the background-color property

jj7lin committed:
implemented DRY principle to code

As per Nadine's request, I reduced repetition of code by "generalizing" a simple button and adding differing/unique properties to differentiate between button 1 from button 2

chalynpai commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,107 @@
+  padding: 0;
+}
+
+li {
+  float: left; /*to get navigation bar items to slide next to each other*/
+}
+
+li a {
+  text-align: center;
+  padding: 14px 16px;
+  display: block;
+  color: black;
+  text-decoration: none;
+}
```
could you merge these two together?

jj7lin replied:
No I don’t think so Cus they’re diff

chalynpai commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,107 @@
+.letsTalkButton {
+  /*"yes, let's match" button */
+  background-color: #e96a6a;
+  border: 2px solid black;
```
add border radius to round the corners and remove the black outline, do this the same for the not interested button please

jj7lin replied:
Okkk

chalynpai commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,107 @@
+  border: 3px solid;
+}
+
+.letsTalkButton:focus {
+  /*when user selects this button, color of button changes*/
+  background-color: #e96a6a;
+  color: white;
+  border: 3px solid;
+}
+
```
whats the difference between focus and hover? the page changes when the user selects a button so is the focus needed? (this is the same for the not interested button

jj7lin replied:
Yeah hover allows the button to change color when your mouse hovers over the button. Focus allows the button to change to a diff color once it has been selected. Both are good visual cues for the user to let her/him know that s/he has done an action

chalynpai commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,107 @@
-    padding: 20px;
-  }
\ No newline at end of file
+  text-align: center;
```
add in font to change the font of "potential friends". Font details are in figma

jj7lin replied:
Okk

chalynpai commented on src/SwipeProfiles.js:
```diff
@@ -1,194 +1,215 @@
+          <text>{error}</text>
+        ) : (
+          <div>
+            <p>Name: </p>
+            <text>{this.state.firstName}</text>
+            <p>Age: </p>
+            <text>{this.state.age}</text>
```
include this section in the profile intro section? it looks a little odd when its separated by the turquoise section

jj7lin replied:
It’s done intentionally - a lot of apps these days separate name and age with the rest of the profile details (ie. tinder). It helps some info stand out more than others, in this case, name and age is what I’d want the user to first see when they view their potential matches

jj7lin committed:
changed font type and added rounded corners to buttons

jj7lin committed:
changed font type and added rounded colors to buttons

As per chalyn's request, I specified the font type to Monteserrat sans-serif and rounded the corners of my buttons so they are no longer rectangular.

jj7lin committed:
Merge branch 'UI_PotentialMatches' of https://github.com/echanlee/PP-friends into UI_PotentialMatches

echanlee commented on src/SwipeProfile.css:
```diff
@@ -1,5 +1,101 @@
+  align-items: center;
+  margin-right: auto;
+  margin-left: auto;
+  width: 500px;
```
it would probably be better to make percentages instead of fixed-width to accommodate for different screen sizes

echanlee commented on src/SwipeProfiles.js:
```diff
@@ -1,194 +1,215 @@
+        });
+    } else {
+      this.setState({
+        error: "There are no potential friends for you",
```
Maybe update message to be more user friendly

jj7lin committed:
changed width from fixed size to percentage and updated "no friends" message

changed width from fixed px to percentage to accomodate for diff screen sizes, and made "no friends" message more user friendly

jj7lin committed:
attempted-unit-testing

attempted to write unit testing to check if buttons are the right colors and also if they change color after a mouse hovers over them. To check test results, run 'npm test start'
